packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -1486,6 +1488,8 @@ curl -v -X POST \
 }' "https://${yourOktaDomain}/api/v1/authn"
 ```
 
+> **Note:** `audience` is a <ApiLifecycle access="deprecated" /> property.
```

Comment:
same comment: is `audience` a parameter or a property?
Also, the developer should provide a new snippet without `audience` if it's deprecated? Is there a reason why we want to keep this here?

Reply:
I'll follow up with the developer but, based on what I see in the related Jira ticket, they don't want to remove the functionality yet. So, technically, they're in the process of deprecating it. [OKTA-385098](https://oktainc.atlassian.net/browse/OKTA-385098)